generators/earlgrey

```diff
@@ -11,6 +11,7 @@ templ = """/*
 :files: {0}
 :incdirs: {1}
 :tags: earlgrey
+:defines: {2}
```

Comment:
That should do!

What is the accepted format of a define?

I think that we should have at least one static/non-generated test that demonstrates this.